1.traverseLinkedList.js

```diff
@@ -18,9 +18,17 @@
 // traverse(linkedlist) should return:
 // [556, 41, 33, 150, 11]
 
-
 function traverse(list) {
+    let result = [];
+    let current = list;
+
+    while(current) {
+        result.push(current.value);
+
+        current = current.next;        
+    }
 
+    return result
 }
```

Comment:
You need to use .val for this problem. Always look to see how the node is constructed.